src/core/client/stream/components/Permalink/Permalink.tsx

```diff
@@ -0,0 +1,38 @@
+import { Localized } from "fluent-react/compat";
+import React from "react";
+import { Button, ButtonIcon, Popover } from "talk-ui/components";
+import PermalinkPopover from "./PermalinkPopover";
+
+interface InnerProps {
+  commentID: string;
+  assetURL: string | null;
+}
+
+class Permalink extends React.Component<InnerProps> {
+  public render() {
+    const { commentID, assetURL } = this.props;
+    return (
+      <Popover
+        placement="top"
+        body={({ toggleVisibility, forwardRef }) => (
+          <PermalinkPopover
+            permalinkUrl={`${assetURL}&commentID=${commentID}`}
+            forwardRef={forwardRef}
```

Comment:
It looks like the ref is already consumed inside the `Popover`, so this won't be needed, see review of the `Popover` component.